src/main/java/cz/cvut/kbss/termit/service/config/ConfigurationProvider.java

```diff
@@ -25,8 +36,14 @@ public ConfigurationProvider(Configuration config) {
      * @return Configuration object
      */
     public ConfigurationDto getConfiguration() {
-        final ConfigurationDto result = new ConfigurationDto();
-        result.setLanguage(config.get(ConfigParam.LANGUAGE));
-        return result;
+        try {
```

Comment:
See the comment on URL/URI as ConfigurationDto identifier. It would get rid of this ugly try-catch.